ompi/instance/instance.c

```diff
@@ -74,6 +76,14 @@ __opal_attribute_constructor__ static void instance_lock_init(void) {
 /** MPI_Init instance */
 ompi_instance_t *ompi_mpi_instance_default = NULL;
 
+/**
+ * @brief: Base timer initialization. All timers returned to the user via MPI_Wtime
+ *         are relative to this timer. Setting it early in during the common
+ *         initialization (world or session model) allows for measuring the cost of
+ *         the MPI initialization.
+ */
+struct timespec ompi_wtime_time_origin = {.tv_sec = 0};
```

Comment:
Should this be initialized to -1 instead of 0, per https://github.com/open-mpi/ompi/issues/12953?

Reply:
We only initialize this base timer once, so we don't really care about the default value. I changed the code to reflect this.